packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -302,12 +302,6 @@ const ModalManager = () => {
         is_mt5_password_invalid_format_modal_visible ||
         is_sent_email_modal_enabled;
 
-    const url_params = new URLSearchParams(useLocation().search);
-    const url_action_param = url_params.get('action');
-    if (url_action_param) {
-        return null;
-    }
-
```

Comment:
This was added to move redundant modals but removing this logic since appstore package have its own modal manger to trigger modals